apps/researcher/src/app/[locale]/objects/[id]/page.tsx

```diff
@@ -25,7 +68,11 @@ export default async function Details({params}: Props) {
     return <div data-testid="no-entity">{t('noEntity')}</div>;
   }
 
-  useCurrentOwner.setState({name: object.owner?.name});
+  let organization = null;
```

Comment:
A recurring question :) Is this a `null` or an `undefined`? `organizations.getById()` returns the latter.

Reply:
This should be undefined.